### PR TITLE
Ss 25 set up portfolio page

### PR DESCRIPTION
**Pull request by nreusken**
The portfolio part should be working beside the fact that the chart is not connected to the portfolio data. The dashboard is not totally finished, missing the news and not the correct top movers but will be working on that later this week.

### CODE REVIEW

**Review by ojas1542 — CHANGES_REQUESTED** [review]
Dashboard and portfolio are looking really good so far. Just a few minor issues:
1) There are some unnecessary console.log statements 
2) Spacing is inconsistent in some places(like with the class components)
3) Some lines of code are too long and should be wrapped

File: frontend/src/components/Dashboard.js [hunk 1]
```diff
@@ -1,10 +1,264 @@
...
+    }
+
+    var performance = 0.0;
+    var worth = 0.0;
+    var capitalInvested = 0;
+    var highestPerforming = [0,0];
+    const portfolioDict = port[0];
+    const portfolio = [];
+    for(var key in portfolioDict, data){
+        portfolio.push(createData(key, portfolioDict[key]['amount'], portfolioDict[key]['shares'], data[key]['latestTrade']['p'], data[key]['dailyBar']['h'], data[key]['dailyBar']['l'], data[key]['prevDailyBar']['c']));
```
Comment (ojas1542) [inline frontend/src/components/Dashboard.js:152]: could wrap this line since it's kinda long.

**Comment by mihika-marathe** [comment]
Is the profile button on the menu and on the drawer intentional?

**Review by mihika-marathe — APPROVED** [review]

**Review by Gujsingh — APPROVED** [review]
Looks good to me

**Review by ojas1542 — CHANGES_REQUESTED** [review]
Still seeing the inconsistent indent spacing in places like Dashboard.js

**Review by ojas1542 — APPROVED** [review]

**Pull request merged**